docker/run.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+echo "Running rsync..."
+rsync --config=/etc/rsyncd.conf --daemon &
```

Comment:
This will work. Alternatively, I've used [s6-overlay](https://github.com/just-containers/s6-overlay) before and that works quite well. I mostly like their "chown everything on startup" thing.